src/ScottPlot/Plot/Plot.Render.cs

```diff
@@ -166,6 +167,20 @@ public Bitmap Render(int width, int height, bool lowQuality = false, double scal
         /// </summary>
         public Bitmap GetBitmap(bool lowQuality = false, double scale = 1.0) => Render(settings.Width, settings.Height, lowQuality, scale);
 
+        /// <summary>
+        /// Render the plot and return the bytes for a PNG file.
+        /// This method is useful for rendering in stateless cloud environments that do not use a traditional filesystem.
+        /// </summary>
+        /// <returns></returns>
+        public byte[] GetImageBytes(bool lowQuality = false, double scale = 1.0)
```

Comment:
It's a little weird that this returns a PNG but it's name makes out like it returns only the body (i.e. doesn't include the header).

Reply:
> it's name makes out like...

I tried to leave this open-ended because I plan to have it accept a `Maui.Graphics.ImageFormat` enum as an argument and perhaps then it will be more obvious what this returns 🤔 

https://github.com/dotnet/Microsoft.Maui.Graphics/blob/0e9a4cdadbc077cc29103c0a12a5b24b5beb6e7e/src/Microsoft.Maui.Graphics/ImageFormat.cs#L1-L11

Reply:
I think that would make more sense for something like `GetImage`, but you're right, taking in that enum would resolve the confusion.